### ISSUE TITLE
How to perform Evaluation Episode with special environment parameters?

### ISSUE THREAD

**Issue by mattiasmar** [NervanaSystems/coach#376]
Is it possible to instruct Coach to use special environment parameters for the evaluation episodes?

**Comment by gal-leibovich** [comment]
Coach supports Gym API and does not extend over it. Do you know if Gym API's supports that?

**Comment by mattiasmar** [comment]
I was thinking that it would be possible to let Coach use one environment instance for training and another instance for evaluation. From a Gym API perspective it is valid.

**Comment by gal-leibovich** [comment]
You're right. Unfortunately, Coach supports interacting only with a single environment.
